hive/hadoop-3.4.0-compat.patch

```diff
@@ -0,0 +1,32 @@
+diff --git a/common/src/java/org/apache/hadoop/hive/common/JvmMetrics.java b/common/src/java/org/apache/hadoop/hive/common/JvmMetrics.java
```

Comment:
Becasuse the `EventCounter` class has been removed since Hadoop v3.4.0 because was using `log4j` 1.x, the following code for JVM metrics no longer compiles.

The following patch removes the event count metric (error, warning, and info logs) from the JVM metrics, that is no longer supported in Hadoop 3.4.0.

The change has been tracked in Hadoop in the issue [HADOOP-17524](https://issues.apache.org/jira/browse/HADOOP-17524), of which the PR [here](https://github.com/apache/hadoop/pull/2909).

Moreover, a related change in Hadoop is tracked [here](https://issues.apache.org/jira/browse/HADOOP-17642).